### DISCUSSION TITLE
Proper Netherite

### DISCUSSION THREAD

**Discussion by kotmatross28729** [Roadhog360/Et-Futurum-Requiem#186] [Improvements]
I would like netherite armor and tools to be created like in vanilla minecraft on a special table, and most importantly, that they retain their enchantments when crafting

**Comment by Omgise** [comment]
I agree with you. :)

**Comment by sousuke0422** [comment]
It has already been implemented and will be included in the next release if there are no problems.
2f3f759107b98902b2ffe8a4dc893fe65827e57d

**Comment by Roadhog360** [comment]
This is false, the smithing table GUI needs more work and will not be included in the next release.

Please do not promise other players content that you did not verify is actually included in the game.
